### ISSUE TITLE
Create illustrations for the Contributors page.

### ISSUE THREAD

**Issue by jlengstorf** [openwhisk/openwhisk.github.io#11]
Currently blocked by #29. Once we get the content, we need the following from @yumeforever:

- [x] Illustration for section 1
- [ ] Illustration for section 2
- [x] Illustration for section 3

(We should update these todos with actual feature/section headlines once we have them.)

**Comment by yumeforever** [comment]
First shot at that along with idea for QUOTE and call to actions in sections. 
![contributor 2x](https://cloud.githubusercontent.com/assets/3612533/20928274/593cc158-bbc5-11e6-8b87-7e09bce24c58.png)

**Comment by jlengstorf** [comment]
@yumeforever We still need the illustration for the service integrators section of this page. See #29 for the content.

**Comment by jlengstorf** [comment]
Opened a separate issue #78 for the final illustration. Closing this.
